01_fib/fibonacci.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import "fmt"
+
+func fib(n int) {
+
+	var (
+		fn int
+		f0 int
+		f1 = 1
+	)
+
+	for i := 1; i <= n; i++ {
+		if i <= 1 {
```

Comment:
Better to initialize the variables so that this edge case isn't needed.

Reply:
yes, fixed this